metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -336,7 +337,11 @@ def _compute_bigquery_lineage_via_gcp_logging(self) -> None:
     def _compute_bigquery_lineage_via_exported_bigquery_audit_metadata(self) -> None:
         logger.info("Populating lineage info via exported GCP audit logs")
         try:
-            _client: BigQueryClient = BigQueryClient(project=self.config.project_id)
+            if self.config.lineage_client_project_id is None:
+                self.config.lineage_client_project_id = self.config.project_id
```

Comment:
I would not overwrite the config value if possible but I would store it in a local variable just to be on the safe side like
```
project_id:str = self.config.lineage_client_project_id if self.config.lineage_client_project_id else self.config.project_id
_client: BigQueryClient = BigQueryClient(
                project=project_id
            )
```